Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Destination/UpdateDestination.cs

```diff
@@ -80,20 +95,39 @@ private static async void updateExistedDocument(dynamic data, ILogger log, BsonD
 
             if (type.Equals(Constants.NOTIFICATION_TYPE_LOCATION))
             {
-                double latitude = Convert.ToDouble(data.location.latitude), longitude = Convert.ToDouble(data.location.longitude);
-                string address = GoogleHttpClient.Instance.GetAddressFromCoordinatesAsync(latitude, longitude, log).Result;
+                if (data.location.longitude == null && data.location.latitude == null)
+                {
+                    document["location"].AsBsonDocument.Remove("latitude");
+                    document["location"].AsBsonDocument.Remove("longitude");
+                    document["location"].AsBsonDocument.Remove("address");
+                }
```

Comment:
What happens if the users don't have the field "location" on this chosen destination?
While trying to remove this document, is that ok or an exception is thrown?